build/yaml/jobs/windows/postbuild.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Job Template: PostBuild steps
+
+parameters:
+  SignType: None
+
+jobs:
+- job: PostBuild
```

Comment:
My rough reading of the requirements for SBOM generation is that this has to contain everything that is publicly shipped. Arguably, this would include all packages whether they are NuGet, MSI, MSM, etc. This probably should be a post-package job and may only need to consider the packages themselves rather than all binaries; the requirements state that only top-level dependencies are required.

Reply:
From the guidance:
```
You should run the generator after you have built your binaries but before you have built your installer manifest and setup packages. The SBOM generator task takes an output location (BuildDropPath) as input and generates an SBOM file for that output. The output directory you specify should contain every file contained the in the packages you are planning on inserting into Visual Studio. 
```

I'll refactor this to pull in only the files we put into our packages.

Reply:
Other guidance that I've seen is `it is okay to generate an SBOM that just lists the NuGet package and not the contents.`, which tells me that the package is what's important, not the individual files within the package. I've sent some email out to hopefully get some clarification on this.